Source/DafnyTests/DafnyTests.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>netcoreapp3.1</TargetFramework>
+        <AssemblyName>DafnyTests</AssemblyName>
+        <GenerateEmbeddedFilesManifest>true</GenerateEmbeddedFilesManifest>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="DiffPlex" Version="1.6.3" />
+        <PackageReference Include="FluentAssertions" Version="5.10.3" />
+        <PackageReference Include="Microsoft.Extensions.FileProviders.Embedded" Version="5.0.0-rc.1.20451.17" />
```

Comment:
Good callout, I don't know why Rider picked a release candidate as the latest version. Upgraded to 5.0.8 now (since it's been over a year since I touched this PR :)